README.md

```diff
@@ -1,15 +1,48 @@
 # AceBook
+## [Makers Academy](http://www.makersacademy.com) - Week 9 Group Project
+Team members:
+* [Merryn Hurley-Rawlins](https://github.com/merrynhr)
+* [Zsuzsanna Ver](https://github.com/MrsVer)
+* [Cynthia Fu](https://github.com/YinnyF)
+* [Fabio Fodrihuez](https://github.com/frodri13)
```

Comment:
small typo here, should be: Fabio Rodriguez

Reply:
TO BE DONE ASAP!!!